idd/Energy+.idd.in

```diff
@@ -67346,7 +67346,7 @@ ThermalStorage:ChilledWater:Stratified,
        \type node
   N13, \field Source Side Heat Transfer Effectiveness
        \type real
-       \minimum 0.0
+       \minimum> 0.0
```

Comment:
I did not see any calculations that the use side being zero caused any problems so that is why I left it alone. I will make the change to the the ThermalStorage:ChilledWater:Mixed.